client/js/components/fragments/Map.js

```diff
@@ -31,11 +31,10 @@ class Map extends Component {
   }
 
   configureMap(google) {
-    this.map = new google.maps.Map(document.getElementById('map'), {
-      center: {lat: 0, lng: 50},
-      zoom: 2,
-      minZoom: 2,
-      mapTypeId: google.maps.MapTypeId.SATELLITE,
+    this.map = new google.maps.Map(document.getElementById('Map'), {
+      center: {lat: 0, lng: 0},
+      zoom: 3,
```

Comment:
Why did you remove the min zoom? When you zoom out to zoom=1 or zoom=1 the app looks bad